source/components/cardContainer/filters/filterGroup/filterGroup.directive.ng1.ts

```diff
@@ -54,5 +68,6 @@ export let filterGroup: angular.IComponentOptions = {
 		icon: '=',
 		filterGroup: '=',
 		source: '=',
+		enabled: '<',
```

Comment:
This is a little nit-picky, but it seems like it's a lot more common in our code-base to allow an option for `disabled` instead of `enabled`. That way it defaults to undefined, which is falsy. Can we just switch it for consistency's sake? Unless you have a reason/preference towards 'enabled'